Libs/DICOM/Widgets/ctkDICOMTableView.cpp

```diff
@@ -146,7 +155,22 @@ void ctkDICOMTableViewPrivate::hideUIDColumns()
 //----------------------------------------------------------------------------
 QString ctkDICOMTableViewPrivate::queryTableName() const
 {
-  return this->lblTableName->text();
+  //return this->lblTableName->text();
+  return this->lblqueryTableName.text();
+}
+
+void ctkDICOMTableViewPrivate::applyHeaderTextList()
+{
+  if (isHeaderTextApplied) return;
+
+  for(int i = 0; i < dicomSQLModel.columnCount(); i++)
+  {
+	QVariant fieldName = dicomSQLModel.headerData(i, Qt::Horizontal, Qt::DisplayRole);
+    QString headerText = QApplication::translate("ctkDICOMTableView", fieldName.toString().toStdString().c_str(), 0);
```

Comment:
STYLE: indentation should match

Reply:
ok, I use sublime text to modify the code, so the indentation is strange, I will modify it .